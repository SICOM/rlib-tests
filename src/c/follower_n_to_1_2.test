COMMAND=./follower_n_to_1_2
FORMATS="pdf xml txt csv html"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
