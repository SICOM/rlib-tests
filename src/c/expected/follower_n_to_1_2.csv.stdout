"First Name          ","Last Name           ","Color               "
"Bob                 ","Doan                ","blue                       NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
"Terry               ","Doan                ","blue                       NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
"Eric                ","Buruschkin          ","green                      NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
"Mike                ","Roth                ","yellow                     NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
"Bob                 ","Kratz               ","pink                       NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
"Steve               ","Tilden              ","purple                     NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
